cmd/alerts/silence/common.go

```diff
@@ -0,0 +1,95 @@
+package silence
+
+import (
+	"bytes"
+	"context"
+	"log"
+
+	"github.com/openshift/backplane-cli/cmd/ocm-backplane/login"
+	"github.com/openshift/backplane-cli/pkg/cli/config"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+const (
+	AccountNamespace = "openshift-monitoring"
+	ContainerName    = "alertmanager"
+	LocalHostUrl     = "http://localhost:9093"
+	PodName          = "alertmanager-main-0"
+)
+
+type logCapture struct {
+	buffer bytes.Buffer
+}
+
+func (capture *logCapture) GetStdOut() string {
+	return capture.buffer.String()
+}
+
+func (capture *logCapture) Write(p []byte) (n int, err error) {
+	a := string(p)
+	_, err = capture.buffer.WriteString(a)
```

Comment:
Is there a reason this is not just using the bytes.Buffer [Write](https://pkg.go.dev/bytes#Buffer.Write) that writes the []bytes directly?